static.config.js

```diff
@@ -459,6 +460,7 @@ const buildPageAtUrl = async (
   returns object with page url, template and data from appropriate query
   */
   const {
+    pageId,
```

Comment:
We need the pageId in order to tell the site_search API which official_document_collection to filter on.